File: strings/709. To Lower Case/toLowerCase.js
/**
 * @param {string} s
 * @return {string}
 */
var toLowerCase = function(s) {
    let result = ""
    for (let i = 0; i < s.length; i++) {
        let c = s[i];
        if (c >= "A" && c <= "Z") {
            c = String.fromCharCode(c.charCodeAt(0) + 32);
        }
        result += c;
    }
    return result;
};

File: strings/709. To Lower Case/toLowerCase.py
class Solution(object):
    def toLowerCase(self, s):
        """
        :type s: str
        :rtype: str
        """
        result = ""
        for c in s:
            if 65 <= ord(c) <= 90:
                result += chr(ord(c) + 32)
            else:
                result += c
        return result